### ISSUE TITLE
Error: undefined method `n_activities' for nil:NilClass

### ISSUE THREAD

**Issue by steko** [JacobEvelyn/friends#44]
```
$ friends suggest
Error: undefined method `n_activities' for nil:NilClass
```

My `friends.md` is very simple since I just added a few activities to test the app.

**Comment by JacobEvelyn** [comment]
Interesting, thanks for the bug report. I'll dig into this later today.

**Comment by changecase** [comment]
I was getting this error when I had few friends listed as well. I can reproduce it while there are fewer than 7 friends.

**Comment by JacobEvelyn** [comment]
This should be fixed now. I'll release a new gem version in a bit. Thanks for the report!

**Comment by JacobEvelyn** [comment]
This is fixed in version 0.7!
